db/primary_migrate/20230814130423_add_last_batch_claimed_at_to_in_person_enrollment.rb

```diff
@@ -1,6 +1,10 @@
 class AddLastBatchClaimedAtToInPersonEnrollment < ActiveRecord::Migration[7.0]
   def change
     add_column :in_person_enrollments, :last_batch_claimed_at, :datetime
-    InPersonEnrollment.update_all("last_batch_claimed_at = status_check_attempted_at")
```

Comment:
This was necessary because `InPersonEnrollment` defines enums that don't exist when the model is called here, resulting in the following exception:

```
Undeclared attribute type for enum 'document_type' in InPersonEnrollment. Enums must be backed by a database column or declared with an explicit type via `attribute`.
```